static/js/chart/draw.ts

```diff
@@ -57,6 +111,15 @@ const AXIS_GRID_FILL = "#999";
 // Max Y value used for y domains for charts that have only 0 values.
 const MAX_Y_FOR_ZERO_CHARTS = 10;
 
+function translate(text: string): string {
+  if (i18next.t(text)) {
+    text = i18next.t(text);
+  }
+  return text;
+}
+function translate_key(key: { label: string; link?: string }) {
```

Comment:
nit: can make this type an interface and have a member function "translate()"

Reply:
I realize this is actually not needed. Now it's just translated